### PR TITLE
Add GitHub configuration files

### PR DESCRIPTION
**Pull request by TommyDL-Infomaniak**

### CODE REVIEW

File: .github/workflows/rebase-default-branch.yml
Comment (sirambd) [inline .github/workflows/rebase-default-branch.yml:1]: It won't work without secrets

**Review by sirambd — CHANGES_REQUESTED** [review]

File: .github/workflows/sentry-release.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.5.0
+      - name: Create Sentry release
+        uses: getsentry/action-release@v1.3.1
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+          SENTRY_PROJECT: "mail-android"
```
[the remark this answers is not in the document]
Comment (TommyDL-Infomaniak) [reply .github/workflows/sentry-release.yml:19]: In the end, Sentry will not be used.

File: .github/workflows/rebase-default-branch.yml
Comment (TommyDL-Infomaniak) [reply .github/workflows/rebase-default-branch.yml:1]: For now, we will not be using this configuration.

File: .github/workflows/sentry-release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (sirambd) [reply .github/workflows/sentry-release.yml]: It must be deleted in this case

**Review by sirambd — APPROVED** [review]

**Pull request merged**